brickflow/codegen/databricks_bundle.py

```diff
@@ -808,9 +808,14 @@ def _build_native_for_each_task(
             depends_on=[],
         )
 
+        if task.for_each_task_conf is None:
+            raise ValueError(
+                f"Error while building for each task {task_name}. "
+                f"Make sure {task_name} has a for_each_task_conf attribute."
+            )
+
```

Comment:
I would just move this validation right after the task type check, no need to create the nested task if for each config are not provided